src/lib/uri/uri.c

```diff
@@ -371,3 +372,92 @@ uri_set_create(struct uri_set *uri_set, const char *str)
 	uri_set->uri_count = 0;
 	return -1;
 }
+
+static unsigned char unreserved_symbols[256] = {0};
+
+/*
+ * Builds an array with unreserved characters.
+ */
```

Comment:
Nit: We prepend functions, structures, fields and so on with documentation comments `/** */`. Applicable here and below.

(Hm, I thought that checkpatch complains on it.)